cluster/gce/util.sh

```diff
@@ -338,14 +338,14 @@ function detect-node-names {
   detect-project
   INSTANCE_GROUPS=()
   INSTANCE_GROUPS+=($(gcloud compute instance-groups managed list \
-    --zone "${ZONE}" --project "${PROJECT}" \
+    --zones "${ZONE}" --project "${PROJECT}" \
     --regexp "${NODE_INSTANCE_PREFIX}-.+" \
     --format='value(instanceGroup)' || true))
   NODE_NAMES=()
   if [[ -n "${INSTANCE_GROUPS[@]:-}" ]]; then
     for group in "${INSTANCE_GROUPS[@]}"; do
       NODE_NAMES+=($(gcloud compute instance-groups managed list-instances \
-        "${group}" --zone "${ZONE}" --project "${PROJECT}" \
+        "${group}" --zones "${ZONE}" --project "${PROJECT}" \
```

Comment:
Yeah, the gcloud release notes aren't very helpful... They say command line flag abbreviations are no longer supported, but don't mention what "command line flag abbreviations" are. Google search doesn't show anything either.